<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Revision</class>
 <widget class="QDialog" name="Revision">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Revision</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget_title">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>新建行</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>气体</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>标称值</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>最小值</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>最大值</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>启用</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>气体</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>标称值</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>最小值</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>最大值</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
         <item row="0" column="4">
          <property name="text">
           <string>启用</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBox_all">
         <property name="text">
          <string>启用</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tableWidget_HC">
       <property name="styleSheet">
        <string notr="true">QTableWidget{
border:1px solid gray;
border-top:none;
border-bottom:none;
}</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>气体</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>标称值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>最小值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>最大值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>启用</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>HC</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>40</string>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item row="1" column="3">
        <property name="text">
         <string>60</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>80</string>
        </property>
       </item>
       <item row="2" column="3">
        <property name="text">
         <string>120</string>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string>160</string>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string>135</string>
        </property>
       </item>
       <item row="3" column="3">
        <property name="text">
         <string>190</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: gray;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_apply_HC">
         <property name="text">
          <string>启用</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: gray;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tableWidget_CO">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>151</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget{
border:1px solid gray;	
border-bottom:none;
}</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>气体</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>标称值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>最小值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>最大值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>启用</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>CO</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>0.4</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>0.5</string>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>0.4</string>
        </property>
       </item>
       <item row="1" column="3">
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item row="2" column="3">
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string>5.2</string>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item row="3" column="3">
        <property name="text">
         <string>6.2</string>
        </property>
       </item>
       <item row="4" column="1">
        <property name="text">
         <string>8.1</string>
        </property>
       </item>
       <item row="4" column="2">
        <property name="text">
         <string>6.7</string>
        </property>
       </item>
       <item row="4" column="3">
        <property name="text">
         <string>9.5</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_apply_CO">
         <property name="text">
          <string>启用</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: gray;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tableWidget_CO2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>151</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget{
border:1px solid gray;	
border-bottom:none;
}</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>气体</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>标称值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>最小值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>最大值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>启用</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>CO2</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>14.9</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>13.1</string>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item row="1" column="3">
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>8.36</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>7.2</string>
        </property>
       </item>
       <item row="2" column="3">
        <property name="text">
         <string>9.2</string>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string>3.27</string>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string>2.5</string>
        </property>
       </item>
       <item row="3" column="3">
        <property name="text">
         <string>4.5</string>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="4" column="1">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="4" column="2">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="4" column="3">
        <property name="text">
         <string>2.5</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_apply_CO2">
         <property name="text">
          <string>启用</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: gray;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tableWidget_NO">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget{
border:1px solid gray;	
}</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>气体</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>标称值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>最小值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>最大值</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>启用</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>NO</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>1000</string>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>800</string>
        </property>
       </item>
       <item row="1" column="3">
        <property name="text">
         <string>1200</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>2500</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>2200</string>
        </property>
       </item>
       <item row="2" column="3">
        <property name="text">
         <string>2800</string>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string>4000</string>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string>3400</string>
        </property>
       </item>
       <item row="3" column="3">
        <property name="text">
         <string>4600</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_apply_NO">
         <property name="text">
          <string>启用</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>1</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: gray;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>59</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_read">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>读取</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
